### PR TITLE
Update Witch_Hunters.cat

### PR DESCRIPTION
**Pull request by loysdeplaisance**
add the link to the prayer choice in warrior priest

### CODE REVIEW

**Comment by loysdeplaisance** [comment]
thanks to you, hope that was helpfull ;)

**Comment by loysdeplaisance** [comment]
Must have a mistak, my android app says "corrrupted file"

**Comment by amis92** [comment]
@loysdeplaisance I think I corrected it. Should work now. Also, wouldn't you like to join our team?

**Comment by loysdeplaisance** [comment]
I Will try it later ;-) 

Why note, but i'm french, and sometime it's an handicap ;-) so if toi can enjoy have à froggy in your tram, that's Will beaucoup an honnor for me

**Comment by amis92** [comment]
Well that'll be just easier for me/us, because instead of doing you forks and patching there, and making pull requests, you'd just push commits to master. However if you don't plan on some bigger activity, it might as well stay as is.

TLDR: If you want to make big changes, ask for invitation. If not, pull requests are good. :)
